<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZincInteractiveDialog</class>
 <widget class="QDialog" name="ZincInteractiveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>186</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="linesSelection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="surfacesSelection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Surfaces</string>
   </property>
  </widget>
  <widget class="QWidget" name="nodeSelectionWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="enableSelection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableCreate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableConstrain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Constrain to surfaces</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nodeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nodes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
